bundles/org.palladiosimulator.dataflow.confidentiality.analysis/src/org/palladiosimulator/dataflow/confidentiality/analysis/sequence/entity/AbstractActionSequenceElement.java

```diff
@@ -8,20 +8,27 @@
 public abstract class AbstractActionSequenceElement<T extends EObject> {
 
     private final Optional<List<DataFlowVariable>> dataFlowVariables;
+    private final Optional<List<CharacteristicValue>> nodeVariables;
```

Comment:
As already mentioned in the single comment, `node characteristics` is more appropriate